mypy/suggestions.py

```diff
@@ -426,7 +430,7 @@ def get_suggestion(self, mod: str, node: FuncDef) -> PyAnnotateSignature:
                 callsites,
                 uses,
             )
-        guesses = self.filter_options(guesses, is_method)
+        guesses = self.filter_options(guesses, is_method, ignore_return=True)
```

Comment:
How is the `ignore_return=True` change related to the rest of the PR?

Reply:
I was abusing `special_form` because `has_any_type` didn't consider it an any. I didn't want to extend it to know about `suggestion_engine` also (and doing so would have interfered some with the use of `suggestion_engine` when refining)